openrouteservice/src/test/java/org/heigit/ors/routing/graphhopper/extensions/storages/builders/WheelchairGraphStorageBuilderTest.java

```diff
@@ -27,12 +27,13 @@ public void reset() {
     @Test
     public void TestProcessSeparateWay() {
```

Comment:
Might also be worth adding a test for when the attribute is tagged as "sidewalk:right:surface" etc.